src/components/Hooks/useSessionstorage.jsx

```diff
@@ -0,0 +1,31 @@
+// TODO Move to cozy-client
+import { useState, useCallback } from 'react'
+
+import log from 'cozy-logger'
+
+export const useSessionstorage = (key, initialValue) => {
+  const [sessionValue, setSessionValueState] = useState(() => {
+    try {
+      const item = window.sessionStorage.getItem(key)
+      return item ? JSON.parse(item) : initialValue
+    } catch (error) {
+      log('error', error)
+      return initialValue
+    }
+  })
+
+  const setSessionValue = useCallback(
+    value => {
+      try {
+        const valueToStore =
+          value instanceof Function ? value(sessionValue) : value
+        setSessionValueState(valueToStore)
+        window.sessionStorage.setItem(key, JSON.stringify(valueToStore))
+      } catch (error) {
+        log('error', error)
+      }
+    },
+    [key, sessionValue]
+  )
+  return [sessionValue, setSessionValue]
+}
```

Comment:
Can you explain the gain of that?What is the difference between that and a cache from cozy-client?

Reply:
We want to keep the contacts added on a paper to offer them for the creation of other paper during the same session.

Reply:
I edited my last commit